integration_tests/src/lib.rs

```diff
@@ -5694,6 +5694,138 @@ mod test {
         assert_eq!(vec![0x55_u8; BLOCK_SIZE * 10], &buffer[..]);
     }
 
+    #[tokio::test]
+    async fn test_volume_replace_rop_in_vcr() {
+        // Verify that we can do replacement of a read_only_parent target
+        // for a volume.
+        const BLOCK_SIZE: usize = 512;
+        let log = csl();
+
+        // Make three downstairs
+        let tds = TestDownstairsSet::small(false).await.unwrap();
+        let opts = tds.opts();
+        let volume_id = Uuid::new_v4();
+
+        let original: VolumeConstructionRequest =
+            VolumeConstructionRequest::Volume {
+                id: volume_id,
+                block_size: BLOCK_SIZE as u64,
+                sub_volumes: vec![VolumeConstructionRequest::Region {
+                    block_size: BLOCK_SIZE as u64,
+                    blocks_per_extent: tds.blocks_per_extent(),
+                    extent_count: tds.extent_count(),
+                    opts: opts.clone(),
+                    gen: 1,
+                }],
+                read_only_parent: None,
+            };
```

Comment:
```suggestion
        let original = VolumeConstructionRequest::Volume {
            id: volume_id,
            block_size: BLOCK_SIZE as u64,
            sub_volumes: vec![VolumeConstructionRequest::Region {
                block_size: BLOCK_SIZE as u64,
                blocks_per_extent: tds.blocks_per_extent(),
                extent_count: tds.extent_count(),
                opts: opts.clone(),
                gen: 1,
            }],
            read_only_parent: None,
        };
```

Reply:
I went through this file and removed all the unneeded `: VolumeConstructionRequest`

Reply:
I also went through `/upstairs/src/volume.rs` and changed all of those.  It added a bunch of noise to the
tests, but there should be no functional change to any of them.